api/src/main/java/io/opentelemetry/trace/propagation/HttpTraceContext.java

```diff
@@ -94,9 +94,19 @@
     chars[0] = VERSION.charAt(0);
     chars[1] = VERSION.charAt(1);
     chars[2] = TRACEPARENT_DELIMITER;
-    spanContext.getTraceId().copyLowerBase16To(chars, TRACE_ID_OFFSET);
+
+    CharSequence traceId = spanContext.getTraceIdAsBase16();
```

Comment:
Looks like a lot can be updated to use `String` / getChars instead of `CharSequence`

Reply:
yeah, it's true. I hadn't tracked them all down. I don't think it's actually harmful to declare these as CharSequence, so I wasn't aggressive at finding them all.

Reply:
I'm going to go through and get rid of the CharSequence usage and go to strictly String based API. We can always loosen it up later, if we just support String for now.

Reply:
done